ledfx_frontend/static/js/904.090f0346.chunk.js

```
@@ -0,0 +1 @@
+(()=>{let e=null;self.onmessage=async function(t){const{action:n,interval:a,scenePL:s,scenePLactiveIndex:c,scenePLrepeat:l,url:o}=t.data;"start"===n?null===e&&(e=setInterval((async()=>{let e=c+1;if(e>=s.length){if(!l)return void self.postMessage({action:"stop"});e=0}const t=s[e];try{await fetch(o+"/api/scenes",{method:"PUT",headers:{"Content-Type":"application/json"},body:JSON.stringify({id:t,action:"activate"})}),self.postMessage({action:"sceneChanged",nextIndex:e})}catch(n){console.error("Failed to activate scene:",n)}}),a)):"stop"===n&&null!==e&&(clearInterval(e),e=null)}})();
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)